src/flags.ts

```diff
@@ -145,39 +145,39 @@ export const help = (opts: Partial<BooleanFlag<boolean>> = {}): BooleanFlag<void
     },
   })
 
-type ElementType<T extends ReadonlyArray<unknown>> = T[number]
+type ReadonlyElementOf<T extends ReadonlyArray<unknown>> = T[number]
```

Comment:
This isn't part of the fix - I just thought it would be a clearer name for the type